codegen/src/test/java/io/improbable/keanu/codegen/python/ParamStringProcessorTest.java

```diff
@@ -0,0 +1,68 @@
+package io.improbable.keanu.codegen.python;
+
+import com.sun.javadoc.ConstructorDoc;
+import com.sun.javadoc.Tag;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import java.util.Map;
+import java.util.Set;
+
+import static io.improbable.keanu.codegen.python.ParamStringProcessor.getNameToCommentMapping;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.IsCollectionContaining.hasItems;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.mockito.Mockito.when;
+
+public class ParamStringProcessorTest {
+
+    private static final String FIRST_PARAM_NAME_CAMEL_CASE = "paramTest";
+    private static final String FIRST_PARAM_NAME_SNAKE_CASE = "param_test";
+    private static final String SECOND_PARAM_NAME_CAMEL_CASE = "anotherParamTest";
+    private static final String SECOND_PARAM_NAME_SNAKE_CASE = "another_param_test";
+    private static final String FIRST_PARAM_COMMENT = "This is a comment about the first parameter paramTest";
+    private static final String SECOND_PARAM_COMMENT = "This is another comment about the second parameter anotherParamTest";
+    private static final String FIRST_PARAM_TEXT = FIRST_PARAM_NAME_CAMEL_CASE + " " + FIRST_PARAM_COMMENT;
+    private static final String SECOND_PARAM_TEXT = SECOND_PARAM_NAME_CAMEL_CASE + "   " + SECOND_PARAM_COMMENT;
+    private static final String RETURN_TAG_TEXT = "this";
+
+    private static final String PARAM_TAG_NAME = "@param";
+    private static final String RETURN_TAG_NAME = "@return";
+
+    @Mock ConstructorDoc constructorDoc;
+    @Mock Tag tag1;
+    @Mock Tag tag2;
+    @Mock Tag tag3;
```

Comment:
`tag3` is never used?

Reply:
Yeah so I guess I was trying to do use this indirectly by checking it doesn't cause another entry to appear in the map. Hence the line `AssertThat(keySet, hasSize(2));`. If you think this is not worth having in the test then I'm happy to remove